packages/eds-core-react/src/components/Button/tokens/icon.ts

```diff
@@ -61,7 +64,7 @@ export const primary: ButtonToken = mergeDeepRight(button, {
     },
     focus: {
       outline: {
-        offset: '0px',
+        offset: `-${parseInt(focusOutlineWidth)}px`,
```

Comment:
This was the fix for having the focus frame "fit" when used inside table?

Reply:
Yes the ghost icon button

Reply:
Its just weird if you have it in focus and hover over it, but who am I kidding. Like thats ever going to be a common use-case 🤪